tools/submission/submission_checker.py

```diff
@@ -776,6 +776,33 @@ def uses_early_stopping(self, scenario):
         return (
             scenario in ["Server", "SingleStream", "MultiStream"]
         )
+    
+    def requires_equal_issue(self, model, division):
+        return (
+            division in ["closed", "network"] and
+            model in [
+                "3d-unet-99",
+                "3d-unet-99.9"
+                "llama2-70b-99",
+                "llama2-70b-99.9"
+                "llama2-70b-99",
+                "llama2-70b-99.9", 
+                "mixtral-8x7b"
+            ]
+        )
+    
+    def guess_equal_issue_mode(self, model):
+        return (
+            self.version in ["v2.1", "v3.0", "v3.1", "v4.0", "v4.1"] and
+            model in [
+                "3d-unet-99",
+                "3d-unet-99.9"
+                "llama2-70b-99",
+                "llama2-70b-99.9"
+                "llama2-70b-99",
```

Comment:
Mixtral?

Reply:
Why is this needed? I don't see if getting used.

Reply:
> Why is this needed? I don't see if getting used.

I think Arjun means it's a typo since "llama2-70b-99" is duplicated.

But yeah, +1 on `guess_equal_issue_mode` function not being called.

Reply:
Fixed